api/src/main/java/io/opentelemetry/baggage/BaggageManager.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
```

Comment:
Nit: 2020

Reply:
I'm ok with merging too since we should address this globally by removing year number

Reply:
This is not a new file, it is moving the old file but github shows it as a new file for whatever reason.